lib/base/Domain.js

```diff
@@ -21,7 +20,9 @@ function Domain(twilio, baseUrl) {
  * @return {string} absolute url
  */
 Domain.prototype.absoluteUrl = function(uri) {
-  return _.trim(this.baseUrl, '/') + '/' + _.trim(uri, '/');
+  const cleanUri = uri.replaceAll('/', '');
+  const cleanBaseUrl = this.baseUrl.replaceAll('/','');
```

Comment:
`replaceAll()` is only supported by Node 15+, which is why Node 14 tests are breaking